zerver/lib/home.py

```diff
@@ -101,6 +101,10 @@ def get_user_permission_info(user_profile: Optional[UserProfile]) -> UserPermiss
         )
 
 
+def get_zulip_user_bots() -> Dict[str, int]:
+    return {email: get_system_bot(email).id for email in settings.CROSS_REALM_BOT_EMAILS}
```

Comment:
System bots are already present in `page_params` -- see `zerver.lib.events.py`, in the `cross_realm_bots`.  So we should be able to avoid any extra queries by using that. (Ends up in `people.js` somewhere).